Templates/AppProject.stencil

```diff
@@ -1,29 +1,31 @@
 import ProjectDescription
-import ProjectDescriptionHelpers
-import MyPlugin
 
-/*
-                +-------------+
-                |             |
-                |     App     | Contains {{ name }} App target and {{ name }} unit-test target
-                |             |
-         +------+-------------+-------+
-         |         depends on         |
-         |                            |
- +----v-----+                   +-----v-----+
- |          |                   |           |
- |   Kit    |                   |     UI    |   Two independent frameworks to share code and start modularising your app
- |          |                   |           |
- +----------+                   +-----------+
-
- */
-
-// MARK: - Project
-
-// Local plugin loaded
-let localHelper = LocalHelper(name: "MyPlugin")
-
-// Creates our project using a helper function defined in ProjectDescriptionHelpers
-let project = Project.app(name: "{{ name }}",
-                          destinations: .{{ platform }},
-                          additionalTargets: ["{{name}}Kit", "{{name}}UI"])
+let project = Project(
+	name: "{{ name }}",
+	targets: [
+		.target(
+			name: "{{ name }}",
+			destinations: .{{ platform }},
+			product: .app,
+			bundleId: "io.tuist.{{ name }}",
+			infoPlist: .extendingDefault(
+                with: [
+                    "UILaunchStoryboardName": "LaunchScreen.storyboard",
+                ]
+            ),
+			sources: ["{{ name }}/Sources/**"],
+			resources: ["{{ name }}/Resources/**"],
+			dependencies: []
+		),
+        .target(
+			name: "{{ name }}Tests",
+			destinations: .{{ platform }},
+			product: .unitTests,
+			bundleId: "io.tuist.{{ name }}Tests",
+			infoPlist: .default,
+			sources: ["{{ name }}/Tests/**"],
+			resources: [],
+			dependencies: [.target(name: "{{ name }}")]
+		),
+	]
+)
```

Comment:
nit: maybe it's just me, but I would indent using 2 spaces

Reply:
Indentation in tuist is 4 spaces, also for all of our fixtures, so I'd keep that

Reply:
Good to me to use 4 spaces, but this is not 4 spaces, it's tabs :D
Also, I think some lines are misaligned as they use spaces instead

Reply:
Yeah, that was there before 🙈 let me update it